src/en/07_ffi.md

```diff
@@ -130,7 +130,14 @@ The following types are considered C-compatible:
 - `repr(C)`-annotated `struct`,
 - `repr(C)` or `repr(Int)`-annotated `enum` with at least one variant and only
   fieldless variants (where `Int` is an integral primitive type),
-- pointers.
+- pointers,
+- an `Option<T>` where `T` is either
+  - `core::ptr::NonNull<U>` and `U` is a `Sized` C-compatible type, then it is
```

Comment:
The part with the pointers (such as `*mut U`) forget to express the `U` is a `Sized` ... invariant that `Option<ptr::NonNull<U>>` do.

I'd weaken the C-compatible bound on `U`, but explicit the `Sized`requirement:

```suggestion
- pointers to some type `Pointee`, where:
  - `Pointee : Sized`,
  - `Pointee` is either a C-compatible type too (and only then can the pointer be dereferenced by the C side), or it is not, in which case it must be treated as a pointer to an opaque object (_c.f._, the dedicated section \[FIXME:add link\]).
- an `Option<T>` where `T` is either
  - `core::ptr::NonNull<Pointee>` where `Pointee` meets the constraints mentioned for the pointer case.
```